docs/stencil-docs/storefront-customization/using-custom-fonts-and-icons.md

```diff
@@ -110,30 +113,30 @@ display: block;
 <!-- theme: {{callout_type}} -->
 
 ### CDN Handlebars
-> The CDN custom Handlebars helper assumes WebDAV's default /content/ folder, so there is no need to specify that folder.
+> The CDN custom Handlebars helper assumes WebDAV's default `/content/` folder, so there is no need to specify that folder.
 > Fonts staged via WebDAV, as in the above example, will not show up in the local version of your theme.
 
 </div>
 </div>
 </div>
 
-Any location where the font is not overridden in the CSS, it falls back on the font in `config.json`. Make sure to update the font there to prevent fallbacks.
+Locations where the font is not overridden in CSS will fall back on the font in `config.json`. Make sure to update the font there to prevent fallbacks.

```

Comment:
```suggestion
The location font will reference the settings in the `config.json` file if the CSS does not override the font in a specific location.
```
This might make the message clearer. Let me know if I'm obscuring the meaning by changing the wording, @ektravel .

Reply:
@uwikaiddi What about this?
Files that do not contain font-overriding CSS code will use the CSS settings in the `config.json` file.

Reply:
@ektravel, awesome!